public/index.html

```diff
@@ -6,7 +6,7 @@
 
 <div class="top-bar">
   <div class="inner">
-    <img src="https://assets.digital.cabinet-office.gov.uk/static/gov.uk_logotype_crown-c09acb07e4d1d5d558f5a0bc53e9e36d.png" width="35" height="31" alt=""> GOV.UK
+    <img src="https://assets.publishing.service.gov.uk/static/images/gov.uk_logotype_crown-ea874a79e09423d63420aff44f016fd0b92dc6dec0cc2668d63b150c8669875e.png" width="35" height="31" alt=""> GOV.UK
```

Comment:
Will this change in the future, when the assets are redeployed?

Reply:
Yes, but _I think_ we rarely rebuild the homepage, and we've already done a bunch of migration work. Shouldn't break for a while.

Reply:
Ok cool 👍